goagen/gen_app/test_generator.go

```diff
@@ -59,9 +60,8 @@ func (g *Generator) generateResourceTest() error {
 		return nil
 	}
 	funcs := template.FuncMap{
-		"isSlice":       isSlice,
-		"initArguments": initArguments,
-		"tempvar":       tempvar,
+		"isSlice": isSlice,
+		"escape":  escape,
```

Comment:
Yes sorry you're correct. `reservedNames` and `escape` should be scoped to `TestMethod` - not `Generator`.